pkg/internal/client/client.go

```diff
@@ -21,16 +21,22 @@ package client
 
 import (
 	"context"
+	"log"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8scorev1 "k8s.io/api/core/v1"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/rest"
 
+	netattdefv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
 	kvcorev1 "kubevirt.io/api/core/v1"
+	netattdefclient "kubevirt.io/client-go/generated/network-attachment-definition-client/clientset/versioned/typed/k8s.cni.cncf.io/v1"
```

Comment:
Please consider using the client from https://github.com/k8snetworkplumbingwg/network-attachment-definition-client.

Reply:
good idea. DONE